src/runtime-rs/crates/hypervisor/src/ch/inner_device.rs

```diff
@@ -125,10 +125,13 @@ impl CloudHypervisorInner {
         Ok(())
     }
 
-    pub(crate) async fn get_shared_fs_devices(&mut self) -> Result<Option<Vec<FsConfig>>> {
+    pub(crate) async fn get_configs_from_pending_devices(
```

Comment:
IMO this function should be separated into two functions, i.e. `get_shared_fs_configs()` and `get_netdev_configs()`.

Merging this two functionalities into one function named `get_configs_from_pending_devices()` is too vague.